addons/sale/models/sale.py

```diff
@@ -182,7 +182,10 @@ def _search_invoice_ids(self, operator, value):
 
     user_id = fields.Many2one(
         'res.users', string='Salesperson', index=True, tracking=2, default=lambda self: self.env.user,
-        domain=lambda self: [('groups_id', 'in', self.env.ref('sales_team.group_sale_salesman').id)])
+        domain=lambda self: "[('groups_id', '=', {}), ('share', '=', False), ('company_ids', '=', company_id)]".format(
```

Comment:
Why don't you keep the "in" for the groups_id part of the domain ? :thinking:

Reply:
```py
('groups_id', '=', 1)
('groups_id', 'in', 1)
('groups_id', 'in', [1])
('groups_id', '=', [1])
```
are equivalent, but I thought the expected syntax was:

```py
('groups_id', '=', 1)
('groups_id', 'in', [1])
```
so it looked ok to me